lib/index.js

```diff
@@ -263,16 +306,98 @@ api.get = async ({actor, account, email, type} = {}) => {
   } else {
     query['account.email'] = email;
   }
+
+  const tokenId = 'meta.bedrock-authn-token.tokens.nonce.id';
+  if(id) {
+    query[tokenId] = id;
+  }
+
   const record = await database.collections.account.findOne(
     query, {projection: {_id: 0, [typeKey]: 1}});
+
+  if(!record) {
+    throw new BedrockError(
+      'Authentication token not found.',
+      'NotFoundError', {
+        httpStatusCode: 404,
+        public: true
+      });
+  }
+
+  let result = record.meta['bedrock-authn-token'].tokens[type];
+  let rValue;
+  if(type === 'nonce') {
```

Comment:
Do we still need this? Can we have token IDs for all tokens?

Reply:
Oh, nevermind, nonces are the only type that can have multiple tokens. Resolving this.